chalice/app.py

```diff
@@ -1950,8 +1950,11 @@ def _unhandled_exception_to_response(self) -> Response:
             body: Any = stack_trace
             headers['Content-Type'] = 'text/plain'
         else:
+            context = getattr(self.current_request, 'lambda_context', None)
+            request_id = getattr(context, 'aws_request_id', None)
```

Comment:
I'd write this even more defensively.

```suggestion
            try:
                request_id = self.current_request.lambda_context.aws_request_id
            except:
                request_id = None
```